books/RayTracingInOneWeekend.html

```diff
@@ -3481,7 +3481,8 @@
 Dividing by the (negative) radius flips the normal as we want. If you implmented your code like the
 second example above, you'll want to fix that now.
 
-Let's use this hollow sphere hack to model a sphere with a given thickness:
+Let's use this hollow sphere hack to model a sphere with a given thickness -- that is, with an
+interior surface. To do this, add a second _inverted_ glass sphere inside the original glass sphere:
```

Comment:
Slight rephrasing suggestion: `Let's use this hollow sphere hack to model the interior surface of a sphere with a given thickness. To do this, add a ...`

Reply:
Done.